recipe/meta.yaml

```diff
@@ -47,14 +47,15 @@ requirements:
     - hdf5
     - fmt >9,<10
     - spdlog =1.12
-    - openssl
+    - openssl >=3.1.2,<3.5
```

Comment:
This is not a fix, in the sense that it will make spiceql impossible to co-install with pretty much all contemporary conda-forge packages. OpenSSL follows SemVer very strictly, so I'd be interested if you encountered any kind of actual breakage?

Reply:
@h-vetinari Right, it's not so much a "fix" but allows spiceql to be installed along side of the current [ISIS](https://github.com/DOI-USGS/ISIS3) dependencies. Given the complexity of the ISIS dependency set it was easier to downgrade the OpenSSL version in spiceql than try to update it the ISIS dependencies